ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -51,6 +51,15 @@ tests:
     make lint
   container:
     from: golangci-lint
+- as: unit
```

Comment:
```suggestion
- as: unit-test
```

Reply:
Was following the same naming scheme as the rest of the configurations. It is true that I would rather add `-test` but it makes this one stand out as the different.

Reply:
Not sure I am following. This is just the name of the prow job, the actual command is in line 55, `make test`.